### DISCUSSION TITLE
`BT-01(d)-Procedure` and `BT-01(f)-Procedure` text or code?

### DISCUSSION THREAD

**Discussion by csnyulas** [OP-TED/eForms-SDK#823] [Q&A] [answered]
Both in case of `BT-01(d)-Procedure` 
![image](https://github.com/OP-TED/eForms-SDK/assets/2441046/4e6be370-cb48-4aaf-8612-951be91dd20d)
https://github.com/OP-TED/eForms-SDK/blob/37ce7b605a14b1babfd0fd5381f48dfbdcf842f3/fields/fields.json#L2070-L2079

and `BT-01(f)-Procedure`
![image](https://github.com/OP-TED/eForms-SDK/assets/2441046/92d8a2a7-f6d4-4923-baba-883d86bc4214)
https://github.com/OP-TED/eForms-SDK/blob/37ce7b605a14b1babfd0fd5381f48dfbdcf842f3/fields/fields.json#L2142-L2151

the fields seem to be about a "description" of the Legal Basis, with a multilingual text as their value (`"type" : "text-multilingual"`), however the `legalType` is "CODE".

This seems to be a mistake. Can you confirm? If yes, are you planning to fix it?

**Comment by YvesJo** [comment, marked as answer]
Hi,
The fields in questions are derived for BT-01 which was exclusively foreseen for the reference to the Standard directives. These additional fields allow to satisfy further requirements with the possibility to  specify some local legal basis. 
The legalType refers in fact to the type associated to the Business Term as defined in the Regulation Annex. The Business Term these additional fields were derived from is BT-01 which is defined in the Annex as being of type Code.
KR

**Comment by csnyulas** [comment]
Are you implying that these fields are exceptions i.e. that in case of these derived fields the legal type does not always apply, and it is OK for the field definition to stay like this?

**Reply by YvesJo** [reply]
There are multiple fields for which the Field Type may differ from the associated Business Term Type (legalType) and this may result from differences between the standard used for mark-up (UBL) and the Regulation Annex. legalType is not to be considered in the implementation.
